src/views/utility/preprocessing.js

```diff
@@ -127,17 +131,22 @@ export function getLinkForSegmentNumbers(language, segmentnr) {
       ? `https://www.tipitaka.org/romn/`
       : `https://suttacentral.net/${rootSegment}/pli/ms#${cleanedSegment}`;
   } else if (language === 'chn') {
+    segmentnr = getSegmentIdFromKey(segmentnr);
     const cleanedSegment = segmentnr.split(':')[0].replace(/_[TX]/, 'n');
     linkText = `http://tripitaka.cbeta.org/${cleanedSegment}`;
+  } else if (language === 'tib') {
+    formattedSegmentNr = FormattedSegment({
+      segment: segmentArrayToString(segmentnr, language),
+      lang: language,
+    });
   }
-
   return linkText
     ? html`
         <a target="_blanc" class="segment-link" href="${linkText}"
-          >${segmentnr}</a
+          >${formattedSegmentNr}</a
```

Comment:
This is not a necessary change. In tibetan you never have an external linkText so this will never happen. I suggest to return this to segmentnr so pali and chinese work here.

Reply:
Changed it now. Please check if you think it is appropiate this way. The display now works correctly for Tibetan and Chinese (new files). Pali and Sanskrit will require more debugging...